sdk/identity/azure-identity/azure/identity/_enums.py

```diff
@@ -0,0 +1,71 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+# pylint:skip-file (avoids crash due to six.with_metaclass https://github.com/PyCQA/astroid/issues/713)
+from enum import Enum
+from six import with_metaclass
+
+from azure.core import CaseInsensitiveEnumMeta
+from msal import ConfidentialClientApplication
+
+
+class RegionalAuthority(with_metaclass(CaseInsensitiveEnumMeta, str, Enum)):
```

Comment:
Just a question: should this pattern be applied to all enums, all new enums, or only some?

Reply:
All enums (see the [guidelines](https://azure.github.io/azure-sdk/python_implementation.html#extensible-enumerations)). CaseInsensitiveEnumMeta is relatively new, so we've shipped many enums without it. Adding it to those should be non-breaking but not urgent.